platform/openide.util/src/org/openide/util/MapFormat.java

```diff
@@ -188,9 +189,14 @@ public String processPattern(String newPattern) throws IllegalArgumentException
 
             if (ridx < 0) {
                 if (exactmatch) {
-                    throw new IllegalArgumentException(
-                        NbBundle.getBundle(MapFormat.class).getString("MSG_UnmatchedBraces") + " " + lidx
-                    );
+                    throw new IllegalArgumentException() {
+                        @Override
+                        public String getMessage() {
+                            // this is ugly- but it allows unit testing
```

Comment:
Why?

Reply:
Executing 
NbBundle.getBundle(MapFormat.class).getString("MSG_UnmatchedBraces") 
throws exception saying thet no bundle has been found. 
so I detered this execution to be able to run tets

better solution is to make sure bundle exists while unit tests are being executed - I don't know how to do it yet